cmd/kind/load/docker-image/docker-image.go

```diff
@@ -93,18 +102,33 @@ func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
 
 	// pick only the user selected nodes and ensure they exist
 	// the default is all nodes unless flags.Nodes is set
-	selectedNodes := nodes
+	candidateNodes := nodes
 	if len(flags.Nodes) > 0 {
-		selectedNodes = []clusternodes.Node{}
+		candidateNodes = []clusternodes.Node{}
 		for _, name := range flags.Nodes {
 			node, ok := nodesByName[name]
 			if !ok {
-				return errors.Errorf("unknown node: %s", name)
+				return errors.Errorf("unknown node: %q", name)
 			}
+			candidateNodes = append(candidateNodes, node)
+		}
+	}
+
+	// pick only the nodes that don't have the image
+	selectedNodes := []clusternodes.Node{}
+	for _, node := range candidateNodes {
+		// TODO: move under pkg/
+		cmdNode := node.Command("crictl", "-r", "/var/run/containerd/containerd.sock", "inspecti", imageName)
```

Comment:
@BenTheElder I'll move this under pkg, what name do you prefer pkg/container/cri?

Reply:
moved to a node method